internal/provider/cloudexport_schema.go

```diff
@@ -72,6 +86,12 @@ func makeAWSSchema(mode schemaMode) *schema.Schema {
 		Computed:    mode == readSingle || mode == readList, // provided by server on read
 		Optional:    mode == create,                         // optionally provided by user on create
 		Description: "Properties specific to Amazon Web Services \"vpc flow logs\" exports",
+		ExactlyOneOf: skipOnReadOneOf(mode, []string{
+			models.CloudProviderAWS,
```

Comment:
Could you explain what is the purpose of that? And why we have 4 values here?
I understand that this is a 1-element list of AWS object, so I don't know how ExactlyOneOf applies to this.

Reply:
As discussed in other channel, this is for validation that there is only one object with provider properties. We need to use separate awsKey constant to make it readable.

Reply:
Done